src/main/flight/gps_rescue.c

```diff
@@ -572,18 +571,25 @@ static void sensorUpdate(void)
     // when there is significant velocity error, increase IMU COG Gain for yaw to a higher value, and reduce max pitch angle
     if (gpsRescueConfig()->rescueGroundspeed) {
         const float rescueGroundspeed = 1000.0f; // in cm/s, fixed 10 m/s groundspeed
-        // sets the slope of the increase in IMU COG Gain as velocity error increases; at 10, we get max CoG gain around 25m/s drift, approx the limit of what we can fix
+        // rescueGroundspeed sets the slope of the increase in IMU COG Gain as velocity error increases
         const float groundspeedErrorRatio = fabsf(rescueState.sensor.groundSpeedCmS - rescueState.sensor.velocityToHomeCmS) / rescueGroundspeed;
         // 0 if groundspeed = velocity to home,
         // 1 if moving sideways at 10m/s but zero home velocity,
         // 2 if moving backwards (away from home) at 10 m/s
         // 4 if moving backwards (away from home) at 20 m/s
+        const float pitchAngleFactor = (gpsRescueAngle[AI_PITCH] > 0.0f) ? gpsRescueAngle[AI_PITCH] / rescueGroundspeed : 0.0f;
```

Comment:
`rescueGroundspeed` makes no sense here (reuse of unrelated value)

Reply:
Will move to separate PR

Reply:
Done